examples/example-server-lib/floating_window_manager.cpp

```diff
@@ -77,7 +78,7 @@ FloatingWindowManagerPolicy::~FloatingWindowManagerPolicy() = default;
 
 bool FloatingWindowManagerPolicy::handle_pointer_event(MirPointerEvent const* event)
 {
-    if (MinimalWindowManager::handle_pointer_event(event))
+    if (MinimalWindowManagerV2::handle_pointer_event(event))
```

Comment:
I wonder if we sholudn't use the `using super = MinimalWindowManagerV2` trick in the definition of `FloatingWindowManagerPolicy` to make all of these `super::handle_pointer_event()` etc.

More a comment than a question, though :)